app/components/ToggleOption/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+*
+* ToggleOption
+*
+*/
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';  // eslint-disable-line import/no-unresolved
+
+function ToggleOption(props) {
+  return (
+    <option value={props.value}>
+      <FormattedMessage {...props.message} />
```

Comment:
Great find, okay I'll add this. I was wondering what the approach should be here.